packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -217,6 +217,13 @@ class RabbitResourceTrackingStartedMessage(RabbitResourceTrackingBaseMessage):
     node_id: NodeID
     node_name: str
 
+    parent_project_id: ProjectID
+    root_parent_project_id: ProjectID
```

Comment:
And what is the difference between these?

Reply:
The same as above